starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,40 +56,36 @@ public void findAll() {
     Assert.assertFalse(actualInventory.isEmpty());
   }
 
+  /**
+   * Test Create method 1:
+   * Check for same names
+   * Check for right # of items
+   */
   @Test
-  // Test for Create -> check # of items
   public void create1() {
-    // Creating two new items to "create"
     Inventory item1 = new Inventory();
     Inventory item2 = new Inventory();
-    item1.setName("item1");
-    item2.setName("item2");
-    item1.setProductType(PRODUCT_TYPE);
-    item2.setProductType(PRODUCT_TYPE);
-    // Call the function you want to test
+    item1.setName(NAME1);
+    item2.setName(NAME2);
     this.inventoryDAO.create(item1);
     this.inventoryDAO.create(item2);
-    // Make sure that the objects in Inventory == 2
-    Assert.assertEquals(2, this.mongoTemplate.findAll(Inventory.class).size());
+    List<Inventory> currInventory = this.inventoryDAO.findAll();
+    Assert.assertEquals(NAME1, currInventory.get(0).getName());
+    Assert.assertEquals(NAME2, currInventory.get(1).getName());
+    Assert.assertEquals(2, this.inventoryDAO.findAll().size());
   }
```

Comment:
Just like before, you really should be able to combine these into one test method. You don't really need multiple tests for this since it's all just one case: normally creating a new Inventory object with no errors.